scripts/create_tables.sql

```diff
@@ -116,36 +204,15 @@ ALTER TABLE IF EXISTS public."Dialog"
     OWNER to admin;
 	
 	
-CREATE TABLE public."Item"
+CREATE TABLE public."Player_Item"
 (
-    id_item BIGSERIAL NOT NULL,
-    name VARCHAR,
-    "HP" INTEGER,
-    strength INTEGER,
-    dexterity INTEGER,
-    intelligence INTEGER,
-    charisma INTEGER,
-    perception INTEGER,
-    action_points INTEGER,
-    effect VARCHAR,
-    base_price INTEGER,
-    description VARCHAR,
-    PRIMARY KEY (id_item)
-);
-
-ALTER TABLE IF EXISTS public."Item"
-    OWNER to admin;
-	
-	
-CREATE TABLE public."Creature_Item"
-(
-    id_creature_item BIGSERIAL NOT NULL,
-    id_creature BIGINT,
+    id_player_item BIGSERIAL NOT NULL,
```

Comment:
Po co dodawać id_player_item? Nie lepiej zrobić złożony klucz główny id_player i id_item ?

Reply:
No teoretycznie można tak zrobić, mogę to zmienić, trochę miejsca się zaoszczędzi zawsze